massa-execution-worker/src/active_history.rs

```diff
@@ -259,4 +260,19 @@ impl ActiveHistory {
             }
         }
     }
+
+    /// Get the execution statuses of operations
+    ///
+    /// # Return Value
+    ///
+    /// A hashmap with
+    /// * the operation id as the key
+    /// * and a bool as the value: true: execution succeeded, false: execution failed
+    pub fn get_op_exec_status(&self) -> HashMap<OperationId, bool> {
+        self.0
+            .iter()
+            .flat_map(|exec_output| exec_output.state_changes.executed_ops_changes.clone())
+            .map(|(op_id, (status, _))| (op_id, status))
+            .collect()
```

Comment:
note: that's a huge array. 5000 tx/slot, 100 active slots = half a million elements